.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     'import/no-named-as-default': 'off',
     '@shopify/images-no-direct-imports': 'off',
     '@shopify/jest/no-snapshots': 'off',
-    'no-console': 'off',
+    'no-console': 2,
```

Comment:
Perhaps using `error` would be more clear as to what this is doing?

Reply:
using error now - good call